### PR TITLE
Update Dockerfile and `compose.yml`

### PR DESCRIPTION
**Pull request by ouziel-slama**

### CODE REVIEW

**Comment by ouziel-slama** [comment]
- Add entrypoint in the Dockerfile
- Use default value when possible

File: simplenode/compose.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ services:
       # - "-chain=test"                       # testnet
       - "-rpcallowip=0.0.0.0/0"
       - "-rpcbind=0.0.0.0"
-      - "-rpcuser=rpc"
+      - "-rpcuser=bitcoinrpc"
```
Comment (adamkrellenstein) [inline simplenode/compose.yml:14]: ahh I wanted to standardize on `rpc:rpc` _everywhere_... can we change it on the other end?

File: simplenode/compose.yml [hunk 2]
```diff
@@ -55,19 +55,12 @@ services:
...
       # "--testnet"                           # testnet
       - "--backend-connect=bitcoind"
-      - "--backend-user=rpc"
-      - "--backend-password=rpc"
       - "--indexd-connect=addrindexrs"
-      - "--rpc-user=rpc"
-      - "--rpc-password=rpc"
       - "--rpc-host=0.0.0.0"
-      - "--log-file=1"
-      - "--api-log-file=1"
```
Comment (adamkrellenstein) [inline simplenode/compose.yml:69]: these are on by default?

**Review by adamkrellenstein — APPROVED** [review]

File: simplenode/compose.yml (hunk 2 above)
Comment (ouziel-slama) [reply simplenode/compose.yml:69]: yep since the v10!

**Pull request merged**